lib/rules/value-keyword-case/index.js

```diff
@@ -164,6 +166,10 @@ const rule = function(expectation, options) {
           return;
         }
 
+        if (ignoreProperties.length > 0 && ignoreProperties.includes(prop)) {
```

Comment:
Please use the `matchesStringOrRegExp` util instead of `.includes`, like we do on [this line](https://github.com/Bilie/stylelint/blob/140f09771b9eb26538537e75469a850313ea617b/lib/rules/value-keyword-case/index.js#L164). This will:

1. Allow the option to accept regular expressions.
2. Address the failing [node@4 tests](https://travis-ci.org/stylelint/stylelint/builds/282262478?utm_source=github_status&utm_medium=notification), as node@4 doesn't support `.includes`.